docs/flow-visibility.md

```diff
@@ -0,0 +1,85 @@
+# Flow Visibility
+## Purpose
+Antrea supports sending IPFIX flow records through a flow exporter. The Elastic
```

Comment:
flow exporter -> flow collector?

Reply:
The code that sends flow records in the Antrea Agent is called the flow exporter.

Reply:
Antrea supports sending records to flow collector through its flow exporter.

Reply:
We plan to merge this document with the flow exporter document: https://github.com/vmware-tanzu/antrea/pull/980
Maybe that will make this clearer.